app/web/features/profile/locales/en.json

```diff
@@ -65,5 +65,12 @@
     "message": "Message",
     "default_about_me_headings": "# Current mission\n<br>\ne.g. I want to meet people and learn...\n<br>\n# Why I use Couchers.org\n<br>\ne.g. I think couch surfing should be free...\n<br>\n# My favorite travel story\n<br>\ne.g. One time I...\n",
     "default_hobbies_headings": "# Art\n<br>\ne.g. Painting, ...\n<br>\n# Books\n<br>\neg. The Cat in the Hat, ...\n<br>\n# Movies\n<br>\ne.g. David Attenborough documentaries, ...\n<br>\n# Music\n<br>\ne.g. The Wiggles, ...\n<br>\n",
-    "default_about_home_headings": "# What I can share with guests\n<br>\ne.g. I can share...\n<br>\n"
+    "default_about_home_headings": "# What I can share with guests\n<br>\ne.g. I can share...\n<br>\n",
```

Comment:
I wonder if it's better to break these up in separate strings and have a function or something in a private namespace that piece them back together to make these more translator friendly.

Reply:
I think they'll manage with some guidance. Don't want to over-complicated at the moment